resources/lang/en/info.php

```diff
@@ -23,7 +23,7 @@
     'user_data' => 'User data',
     'workshop' => 'Workshop',
     'workshop_must_be_filled' => 'The workshop must be filled.',
-    'year_of_acceptance' => 'Year of acceptance',
```

Comment:
Most of these are also in the user lang file. I think it would make sense to merge this. Just copy the contents of the file, run sort and remove the dups, then replace `@lang('info.*')` with `@lang('user.*')`. (Can be added as a separate issue.)